docs/Developers/Development_Setup.rst

```diff
@@ -11,10 +11,46 @@ Virtualenv
 
 We highly recommend installing coala in a virtualenv for development. This
 will allow you to have a contained environment in which to modify coala,
-separate from any other installation of coala that you may not want to
-break. For more information about virtualenvs, please refer to the
-`virtualenv setup <https://docs.coala.io/en/latest/Help/MAC_Hints.html#create-virtual-environments-with-pyvenv>`__ section for information on setting one
-up.
+separate from any other installation of coala that you may not want to break.
```

Comment:
How about adding something alog of "This way, you could have 2 separate `coala` instance. Normal coala you get from `pip3 install coala` and the one you compile yourself"? Just to emphasize the function of "virtual environment".